documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -196,6 +196,14 @@ The value of `databaseName` serves as the message key.
 The `tableChanges` field contains an array that includes entries for each column of the table.
 Because the structured representation presents data in JSON or Avro format, consumers can easily read messages without first processing them through a DDL parser.
 
+[IMPORTANT]
+====
+By default, the connector uses the `ALL_TABLES` database view to resolve the table names that should be stored in the schema history topic based on the user account's granted permissions.
```

Comment:
```suggestion
By default, the connector uses the `ALL_TABLES` database view to identify the table names to store in the schema history topic.
Within that view, the connector can access data only from tables that are available to the user account through which it connects to the database.
```
There's a lot packed into this sentence. I think we can make it clearer by breaking out some of the content.
I'm not sure how to interpret `resolve` in this context. I took it to mean _to determine_ or _identify_. But I'm not sure that's right.
When you say `based on the user account's granted permissions` I _think_ that's referring to the permissions granted to the account through which Debezium accesses the database?